bench/src/bench/scala/com/spotify/tf/example/derive/bench/ExampleConverterBenchmark.scala

```diff
@@ -56,7 +56,7 @@ object ExampleConverterBenchmark extends Bench.LocalTime {
 
     measure method "fromExample" in {
       using(exampleGen) in { examples =>
-        examples.foreach(e => converter.fromExample(e))
+        examples.foreach(e => converter.fromExample(e).get)
```

Comment:
Would be good to eliminate this `.get`. It's always a code-smell in scala and looks like we're not using the value anyway.

Reply:
Yep, definitely not needed here 👍